config/aaa.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python -u
+# -*- coding: utf-8 -*-
+
+import click
+import netaddr
+from swsssdk import ConfigDBConnector
+
+
+def is_ipaddress(val):
+    if not val:
+        return False
+    try:
+        netaddr.IPAddress(str(val))
+    except:
+        return False
+    return True
+
+
+def add_table_kv(table, entry, key, val):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.mod_entry(table, entry, {key:val})
+
+
+def del_table_key(table, entry, key):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    data = config_db.get_entry(table, entry)
+    if data:
+        if key in data:
+            del data[key]
+        config_db.set_entry(table, entry, data)
+
+
+@click.group()
+def aaa():
+    """AAA command line"""
+    pass
+
+
+# cmd: aaa authentication
+@click.group()
+def authentication():
+    """User authentication"""
+    pass
+aaa.add_command(authentication)
+
+
+# cmd: aaa authentication failthrough
+@click.command()
+@click.argument('option', type=click.Choice(["enable", "disable", "default"]))
+def failthrough(option):
+    """Allow AAA fail-through [enable | disable | default]"""
```

Comment:
what is the default value here? enable/disable?

Reply:
```
gulv@str-s6000-on-4:~$ show aaa
AAA authentication login tacacs+,local
```
when it is set to default, what is the actual value? enable or disable? same question for failthrough

Reply:
The default value means there is no value for this configuration in configdb. For AAA authentication, the default is local authentication. For AAA fail-through, the default is enable.
I will add all the default value in the show command.